php_phongo.c

```diff
@@ -2214,11 +2214,13 @@ static void php_phongo_command_failed(const mongoc_apm_command_failed_t* event)
 	p_event->operation_id    = mongoc_apm_command_failed_get_operation_id(event);
 	p_event->request_id      = mongoc_apm_command_failed_get_request_id(event);
 	p_event->duration_micros = mongoc_apm_command_failed_get_duration(event);
+	p_event->reply           = bson_copy(mongoc_apm_command_failed_get_reply(event));
```

Comment:
The build currently fails because `mongoc_apm_command_failed_get_reply()` does not exist in the libmongoc submodule. You'll need to do the following to address this:

 * Bump the libbson submodules to its master branch (or the commit where this function first shows up). The PHPC commit for this can use this issue number. For prior art, see: https://github.com/mongodb/mongo-php-driver/commit/a49c71cfc92bf4424adecff74fff1455178b2cb8#diff-25d902c24283ab8cfbac54dfa101ad31
 * In `config.m4`, you'll also need to bump lines that check the version of libmongoc when used as a system library. See: https://github.com/mongodb/mongo-php-driver/blob/94c7487ead392ea655b0fba8a2484c06c617add3/config.m4#L289

Note that libmongoc's master branch (1.10-dev at the moment) incorporates libbson, so I'm not suggesting you bump the libbson submodule since there is nothing to bump to. I assume libmongoc 1.10-dev will still play nice with libbson 1.9 (for now), but if only updating libmongoc causes issues, we can work through a solution in more detail.